scout/server/blueprints/panels/views.py

```diff
@@ -300,7 +317,7 @@ def panel_export_case_hits(panel_id):
         )
         return redirect(request.referrer)
     data = controllers.panel_export_case_hits(panel_id, institute_obj, case_obj)
-    now = datetime.datetime.now().strftime(DATETIME_FORMATTER)
+    now = datetime.datetime.now().strftime(DATE_DAY_FORMATTER)
```

Comment:
Here: you store `now` for the `report_created_at`. Not that it matters to the users when microseconds are not shown. And you are perhaps going to use the filename function for it as well!

Reply:
Ahh yes, but it's a very log and user-unfriendly string I think

Reply:
Ah yes